markdown/blog/helsinki-and-community.md

```diff
@@ -0,0 +1,87 @@
+
+In 2023, we organized multiple events under the brand `AsyncAPI Conf on Tour`. One of them took place at [APIDays Paris](https://www.apidays.global/paris2023/), made possible by the amazing folks behind [APIDays](https://www.apidays.global/), who share our belief in the future of strong API specifications and standards.
+
+It was a fantastic opportunity, but I had one major concern—how do we find our community members in an event with thousands of attendees? How do we have meaningful conversations? And how do we engage with people who don’t yet know about AsyncAPI? As you probably know, conference talks end, and people scatter. It’s easy to lose them in the crowd.
+
+One of the best places to meet people at conferences is at the sponsor booths. But as an open-source, community-driven project, we don’t have the budget for that kind of setup.
+
+Once again, [APIDays](https://www.apidays.global/) came to the rescue and offered us a booth.
+
+We decided to take it one step further—not just a booth for AsyncAPI, but a shared space for other open-source communities. At Paris 2023, we organized the first `API Specs & Standards` booth, bringing together experts from AsyncAPI, OpenAPI, and JSON Schema. We created a vendor-neutral, safe space for knowledge exchange, where experts from different companies shared their expertise without pushing for their products.
+
+## AsyncAPI Conf on Tour at APIDays Helsinki 2024
+
+Our next stop was [APIDays Helsinki 2024](https://www.apidays.global/helsinki_and_north/).
+
+Hugo kicked things off with a great introduction to the AsyncAPI spec and later wrapped up the track with an excellent session on testing event-driven architectures.
+
+<Figure
+  src="/img/posts/helsinki-and-community/hugo.webp"
+  caption="Hugo presenting AsyncAPI basics."
+  className="text-center"
+/>
+
+Jonas followed with a detailed overview of the AsyncAPI tooling ecosystem, showcasing the various tools built around the specification to support developers working with AsyncAPI documents.
+
+<Figure
+  src="/img/posts/helsinki-and-community/jonas.webp"
+  caption="Jonas presenting the AsyncAPI tooling ecosystem."
+  className="text-center"
+/>
+
+I then presented an overview of AsyncAPI use cases, demonstrating how different companies are using AsyncAPI in production, all based on real data we’ve gathered over the years. You can explore all of this in our [AsyncAPI case studies](/casestudies).
+
+<Figure
+  src="/img/posts/helsinki-and-community/lukasz.webp"
+  caption="Lukasz presenting how AsyncAPI is used in production."
+  className="text-center"
+/>
+
+The talks were engaging, and the questions that followed were great. But the best conversations, the most technical and insightful ones, happened at the `API Specs and Standards` booth. We teamed up with friends from the OpenAPI Initiative, and together we had deep, meaningful discussions.
```

Comment:
How about adding details on where to find the talk slides?

Reply:
sure thing

Reply:
but where should I link? it ain't here yet https://conference.asyncapi.com/venue/Helsinki

Reply:
get the link from the Resources Hub

![image](https://github.com/user-attachments/assets/2f51e2c2-4e8c-47c6-931b-b3c9118589fb)

Reply:
oh, ok, would be great to have a dedicated button on Helsinki landing page

anyway, adding a link to https://drive.google.com/drive/folders/1nY7dZF8WFXZ3r2rCWJDDoT2C_GMfQJMV

Reply:
@thulieblack you can have a final look